### PR TITLE
FE-217 - Add principal physician section

### PR DESCRIPTION
**Pull request by lsmartinez88**
# Description

We need to add a new section in the site detail page for the principal physician of a site study. This section could present more than one physician. We are waiting for designers to define how we will show the data on each scenario.

# Jira tickets

Please include a list with the tickets involved in this pull request :pray:

- [FE-217| Add principal physician section](https://trialtech.atlassian.net/browse/FE-217)

# Screenshots

image

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) :bug:
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) :bomb::boom:
- [ ] Configuration / Setup :wrench:
- [ ] Clean up / Technical debt :hammer:

## Apps affected by this change

- [x] Web
- [ ] Landing
- [ ] Management portal

## Project code modified by this change

- [x] Web
- [ ] Landing
- [ ] Management portal
- [ ] Packages

# Checklist:

- [x] I have performed a self-review of my own code :innocent:
- [x] My code is clean and it follows the guidelines of this project :nail_care:
- [x] My changes don't generate new warnings :ok_hand:

### CODE REVIEW

File: apps/web/app/[lang]/sites/[id]/page.tsx [hunk 1]
```diff
@@ -92,6 +88,47 @@ const Page = async ({ params: { lang = defaultLocale, id } }: SiteProps) => {
...
+                        className="flex-col justify-start items-start gap-6 inline-flex"
+                      >
+                        <div className="self-stretch justify-start items-center gap-4 inline-flex">
+                          <Avatar
+                            size="lg"
+                            imageUrl={py.avatar}
+                            alt={`${py.last_name} ${py.first_name}`}
+                          />
+                          <div className="flex-col justify-start items-start gap-1 inline-flex">
+                            <p className="text-primary-dark text-sm md:text-base font-normal font-['DMSans'] uppercase leading-tight">
```
Comment (libattistella) [inline apps/web/app/[lang]/sites/[id]/page.tsx:114]: We can use a Typography here, maybe P2?

File: apps/web/app/[lang]/sites/[id]/page.tsx [hunk 2]
```diff
@@ -92,6 +88,47 @@ const Page = async ({ params: { lang = defaultLocale, id } }: SiteProps) => {
...
+                              {t('principalInvestigator')}
+                            </p>
+                            <p className="text-primary text-xl md:text-2xl font-semibold font-['Lexend'] leading-7">{`${py.first_name} ${py.last_name}`}</p>
```
Comment (libattistella) [inline apps/web/app/[lang]/sites/[id]/page.tsx:117]: Same here

File: apps/web/app/[lang]/sites/[id]/page.tsx [hunk 3]
```diff
@@ -92,6 +88,47 @@ const Page = async ({ params: { lang = defaultLocale, id } }: SiteProps) => {
...
+                          </div>
+                        </div>
+                        <div className="self-stretch flex-col justify-start items-start gap-3 flex">
+                          <div className=" w-full text-dark text-sm md:text-base font-normal font-['DMSans'] leading-normal">
+                            {py.description}
```
Comment (libattistella) [inline apps/web/app/[lang]/sites/[id]/page.tsx:122]: Same

**Review by libattistella — CHANGES_REQUESTED** [review]

**Comment by lsmartinez88** [comment]
Pr changes applied @libattistella

**Review by libattistella — APPROVED** [review]

**Pull request merged**